DashWallet/Sources/UI/CrowdNode/New Account/NewAccountViewController.swift

```diff
@@ -58,44 +61,81 @@ final class NewAccountViewController: UIViewController {
 extension NewAccountViewController {
     private func configureHierarchy() {
         definesPresentationContext = true
-        view.backgroundColor = UIColor.dw_secondaryBackground()
+        view.backgroundColor = UIColor.dw_background()
         
+        configureActionButton()
+        configureAccountAddress()
+        configureTermsCheckBox()
+    }
+    
+    private func configureActionButton() {
         if viewModel.isInterrupted {
             actionButton.setTitle(NSLocalizedString("Accept Terms Of Use", comment: ""), for: .normal)
         } else {
-            actionButton.setTitle(NSLocalizedString("Sign up to CrowdNode", comment: ""), for: .normal)
+            actionButton.setTitle(NSLocalizedString("Create Account", comment: ""), for: .normal)
         }
     }
+    
+    private func configureAccountAddress() {
+        let gradientMaskLayer = CAGradientLayer()
+        gradientMaskLayer.frame = addressLabel.bounds
+        gradientMaskLayer.colors = [UIColor.white.cgColor, UIColor.white.cgColor, UIColor.clear.cgColor, UIColor.clear.cgColor]
+        gradientMaskLayer.locations = [0, 0.7, 0.85, 1]
+        gradientMaskLayer.startPoint = CGPoint(x: 0, y: 0.5)
+        gradientMaskLayer.endPoint = CGPoint(x: 1, y: 0.5)
+        addressLabel.layer.mask = gradientMaskLayer
+    }
+    
+    private func configureTermsCheckBox() {
+        let baseString = NSMutableAttributedString(string: NSLocalizedString("I agree to CrowdNode", comment: "").description)
+        let termsOfUseString = NSMutableAttributedString(string: NSLocalizedString(" Terms of Use ", comment: "").description)
+        let andString = NSMutableAttributedString(string: NSLocalizedString("and", comment: "").description)
+        let privacyPolicyString = NSMutableAttributedString(string: NSLocalizedString(" Privacy Policy ", comment: "").description)
+        
+        termsOfUseString.addAttribute(.link, value: CrowdNodeConstants.termsOfUseUrl, range: NSRange(location: 0, length: termsOfUseString.length))
+        privacyPolicyString.addAttribute(.link, value: CrowdNodeConstants.privacyPolicyUrl, range: NSRange(location: 0, length: privacyPolicyString.length))
+        
+        baseString.append(termsOfUseString)
+        baseString.append(andString)
+        baseString.append(privacyPolicyString)
+        
+        acceptTermsText.attributedText = baseString
```

Comment:
Not sure if there is a better way of having links in text for different languages.